donate/utility/acoustic/acoustic.py

```diff
@@ -115,7 +117,7 @@ def _call_xt(self, xml):
 
     def send_mail(self, to, campaign_id, fields=None, bcc=None, save_to_db=False):
         # If we are testing, do not send emails
-        if not hasattr(settings, "IS_TESTING"):
+        if not TESTING:
```

Comment:
```suggestion
        if not settings.TESTING:
```

Side note: I saw the previous version used `hasattr()`. For some extended reading, check out https://hynek.me/articles/hasattr/

Reply:
Hi Ben,

Thanks for the great feedback and reading material 👍, I have made the requested changes and re-requested review!